tests/integration/src/test/resources/foreach-test.xml

```diff
@@ -51,9 +51,7 @@
     <flow name="parent-message-config">
         <foreach rootMessageVariableName="parent">
             <test:component/>
-            <message-properties-transformer scope="invocation">
```

Comment:
Makes sense to maintain message-properties-transformer or it can be also removed as part of this change?